src/dtb.cpp

```diff
@@ -161,6 +161,16 @@ void dtb_init(const machine_config &c, unsigned char *dtb_start, uint64_t dtb_le
                 fdt.prop_u32_list<2>("interrupts-extended", {PLIC_PHANDLE, plic_irq_id});
                 fdt.end_node();
             }
+            if (c.htif.console_getchar) { // virtio net
```

Comment:
Also the logic will change, right?

Reply:
You commented in an outdated commit, the logic was changed.